pkg/config/samples/default.dusk.toml

```diff
@@ -3,6 +3,7 @@
 # general node configs
 [general]
 network = "testnet"
+walletonly = false
```

Comment:
Could we add some comments explaining what `walletonly` mode means for the node?

Reply:
Yep, will do